src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -0,0 +1,79 @@
+import {Logger} from '@salesforce/core';
+import {Controller} from '../../Controller';
+import {Config} from '../util/Config';
+import {RuleEngine} from '../services/RuleEngine';
+import {Catalog, Rule, RuleGroup, RuleResult, RuleTarget} from '../../types';
+import {ENGINE} from '../../Constants';
+//import childProcess = require('child_process');
+//import path = require('path');
+
+// Unlike the other engines we use, RetireJS doesn't really have "rules" per se. So we sorta have to synthesize a
+// "catalog" out of RetireJS's normal behavior.
+const retireJsCatalog: Catalog = {
+	rules: [{
+		engine: ENGINE.RETIRE_JS.valueOf(),
+		sourcepackage: ENGINE.RETIRE_JS.valueOf(),
+		// Give this rule an informative name, specific enough that we're able to supplement it with other rules later.
+		name: ' insecure-bundled-dependencies',
```

Comment:
Is the `[space]` at the beginning of the name intentional?

Reply:
No, it's a mistake. I've fixed it in the branch that implements the`run` command though, so we don't need to fix it here.